macros/models/_sector/nft/nft_wash_trades.sql

```diff
@@ -155,6 +138,7 @@ SELECT nftt.blockchain
 , nftt.tx_to
 , nftt.block_time
 , date_trunc('day', nftt.block_time) AS block_date
+, date_trunc('month', nftt.block_time) AS block_month
```

Comment:
first attempt at fixing the CI error:
this field needs cast to date, since it's used as the partition column in all spells. might make sense to cast `block_date` too, while you're at it. i know we've seen different error outputs for this in the past, but it's possible the error message changed or something. let's try this first